cadc-pkg-server/src/main/java/org/opencadc/pkg/server/PackageWriter.java

```diff
@@ -88,6 +93,7 @@ public abstract class PackageWriter {
     private static final Logger log = Logger.getLogger(PackageWriter.class);
 
     ArchiveOutputStream aout;
+    boolean openEntry;
```

Comment:
Having this attribute outside the scope of the method in which it is used makes the class not thread safe.

Reply:
ok